main.go

```diff
@@ -194,8 +194,9 @@ func main() {
 	// create a http server
 	server := &http.Server{}
 
-	// create a context that is cancelled on SIGKILL
-	ctx, cancel := context.WithCancel(context.Background())
+	// create a context that is cancelled on SIGKILL or SIGINT
```

Comment:
Minor nit, update comment style.

```suggestion
	// Create a context that is cancelled on SIGKILL or SIGINT.
```